resources/js/features/games/components/AchievementSetEmptyState/AchievementSetEmptyState.tsx

```diff
@@ -44,7 +44,7 @@ export const AchievementSetEmptyState: FC = () => {
               }}
               components={{
                 // eslint-disable-next-line jsx-a11y/anchor-has-content -- this is passed in by the consumer
-                1: auth?.user ? <a href={`/setRequestors.php?g=${backingGame.id}`} /> : <span />,
+                1: auth?.user ? <a href={`/game/${backingGame.id}/requests`} /> : <span />,
```

Comment:
Consistency: We'll want to use `route()` from `ziggy-js` here. When migrating stuff away from PHP, we have the added benefit of being able to leverage strongly-typed routes.

The route definitions can be regenerated by executing the `composer types` command in your terminal.